### PR TITLE
General visual adjustments

### PR DESCRIPTION
**Pull request by ingridklepsvik**
made som visual adjustments to the orgchart and more

### CODE REVIEW

File: src/components/general/IconButton/styles.less [hunk 1]
```diff
@@ -105,7 +105,7 @@
                 height: calc(var(--grid-unit) * 4px);
                 border-radius: 4px;
                 background-color: var(--color-primary-alt3);
-                box-shadow: 0 0 0 2px var(--color-primary-alt3);
+             /*    box-shadow: 0 0 0 2px var(--color-primary-alt3); */
```
[the remark this answers is not in the document]
Comment (ingridklepsvik) [reply src/components/general/IconButton/styles.less]: Yes, removed now

File: src/components/general/IconButton/styles.less [hunk 2]
```diff
@@ -94,7 +94,7 @@
         .iconContainer {
             background-color: var(--color-primary-alt4);
             border-color: var(--color-primary-alt4);
-            box-shadow: 0 0 0 2px var(--color-primary-alt4);
+         /*    box-shadow: 0 0 0 2px var(--color-primary-alt4); */
```
[the remark this answers is not in the document]
Comment (ingridklepsvik) [reply src/components/general/IconButton/styles.less]: Yes, removed now

File: src/components/org/OrgChart/OrgChart.stories.tsx [hunk 3]
```diff
@@ -117,20 +117,42 @@ const positions: Position[] = [
...
+const breadCrumbStyle = {
+    display: 'flex',
+    justifyContent: 'left',
+    alignItems: 'center',
+    backgroundColor: 'white',
+    boxShadow:
+        '0px 1px 5px rgba(0, 0, 0, 0.2), 0px 3px 4px rgba(0, 0, 0, 0.12), 0px 2px 4px rgba(0, 0, 0, 0.14)',
+    margin: '10px',
+    paddingLeft: '16px',
+    paddingRight: '16px',
```
[the remark this answers is not in the document]
Comment (ingridklepsvik) [reply src/components/org/OrgChart/OrgChart.stories.tsx]: Reomoved some of the styling on the breadcrumbs now @eskil0312

File: src/components/org/OrgChart/OrgChart.stories.tsx [hunk 4]
```diff
@@ -179,20 +179,39 @@ const position: Position = {
 };
 const cardStyle = {
```
Comment (eskil0312) [inline src/components/org/OrgChart/OrgChart.stories.tsx]: Is cardStyle used, or could we get rid of it?

File: src/components/org/OrgChart/OrgChart.stories.tsx [hunk 5]
```diff
@@ -231,7 +250,7 @@ const OrgChartStory = () => {
     const componentDisplayType = useComponentDisplayType();
     const cardHeight = componentDisplayType === 'Compact' ? 110 : 142;
     const rowMargin = componentDisplayType === 'Compact' ? 120 : 164;
-    const cardMargin = componentDisplayType === 'Compact' ? 24 : 32;
+    const cardMargin = componentDisplayType === 'Compact' ? 24 : 16;
```
Comment (eskil0312) [inline src/components/org/OrgChart/OrgChart.stories.tsx]: Could we leave this as 32? It's whats being used in the pro-org app

File: src/components/org/OrgChart/components/Aside.tsx [hunk 6]
```diff
@@ -69,7 +69,7 @@ function Aside<T>() {
                     key={card.id}
                     node={card}
                     x={startX + i * (cardWidth + cardMargin)}
-                    y={(rowNo + 1) * (rowMargin - 20) + 20}
+                    y={(rowNo + 1) * rowMargin + 24}
```
Comment (eskil0312) [inline src/components/org/OrgChart/components/Aside.tsx]: Think it's best to use  y={(rowNo + 1) * (rowMargin - 20) + 20} for now, so the spacing between the aside rows don't get too big. Maybe change "+ 20" to "+24"?

File: src/components/org/OrgChart/components/Labels.tsx [hunk 7]
```diff
@@ -59,7 +59,7 @@ const Labels = () => {
             data: asideLabel,
             id: asideLabel,
             x: firstAsideNode && firstAsideNode.x !== null && firstAsideNode.x - 15,
-            y: firstAsideNode && firstAsideNode.y !== null && firstAsideNode.y - 8,
+            y: firstAsideNode && firstAsideNode.y !== null && firstAsideNode.y - 16,
```
Comment (eskil0312) [inline src/components/org/OrgChart/components/Labels.tsx]: Do we need this much spacing between the aside component and the label?

File: src/components/org/OrgChart/OrgChart.stories.tsx (hunk 4 above)
Comment (ingridklepsvik) [reply src/components/org/OrgChart/OrgChart.stories.tsx]: Removed

File: src/components/org/OrgChart/OrgChart.stories.tsx (hunk 5 above)
Comment (ingridklepsvik) [reply src/components/org/OrgChart/OrgChart.stories.tsx]: Changed back to 32

File: src/components/org/OrgChart/components/Aside.tsx (hunk 6 above)
Comment (ingridklepsvik) [reply src/components/org/OrgChart/components/Aside.tsx]: its now "(rowMargin - 20) + 24"

**Pull request merged**